BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/validation/eutoxrisk/services/EUToxRiskFileValidatorService.java

```diff
@@ -38,7 +43,22 @@ public EUToxRiskFileValidatorService(
         this.properties = properties;
     }
 
-    public Collection<EUToxRiskFileValidationError> validate(final File file) {
+    public boolean appliesToProjectId(String accno) {
+        return properties.isEnabled() && properties.getProjectId().equals(accno);
+    }
+
+    public Collection<EUToxRiskFileValidationError> validateFirst(Collection<File> files) {
```

Comment:
I think this should be a List, as there is not a "first" element in a collection as it may be not even ordered (i.e Set is a Collection). So a List will represent the correct abstraction imho.

Reply:
yep.. good catch!